### DISCUSSION TITLE
Forgot Password

### DISCUSSION THREAD

**Discussion by okynnor** [NginxProxyManager/nginx-proxy-manager#2226] [Q&A]
Hi, there, 

I am wondering if there's another way to reset a forgotten password without losing the proxies that were already setup prior? 

I've already tried[ this method](https://github.com/NginxProxyManager/nginx-proxy-manager/issues/230#issuecomment-550565639).  It didn't work as expected. 

Thanks.

**Comment by ClearSEM** [comment]
https://github.com/NginxProxyManager/nginx-proxy-manager/issues/230#issuecomment-859944477
